Create SpecifiedTradeProductXML

BeedleEiEi opened the pull request.

Commit:
Add jacksonXML to dependency

Commit:
Create SpecifiedTradeProductXML

Commit:
Create SpecifiedTradeProductXMLTests

champillon reviewed (commented):
เอา PR ให้มันใหญ่ เห็น combination ของ XML มากกว่านี้หน่อยได้ป่าว

BeedleEiEi commented:
ได้ครับ

Commit:
Merge remote-tracking branch 'refs/remotes/origin/master' into Create-SpecifiedTradeProductXML

Commit:
Merge remote-tracking branch 'refs/remotes/origin/master' into Create-SpecifiedTradeProductXML

Commit:
Add mapping /generate controller

This will get value from the form to controller when select XML

Commit:
Add test in controller

Add test for mapping /generate to get XML buy need to implement XML
converter to be completed.

Commit:
Add /generate/specifiedTradeProductXML in controller

Commit:
Add path /generate to html

Commit:
Change example return text

//TODO implement xml

Commit:
Remove output log

Commit:
Add test for /generate/specifiedTradeProduct

champillon commented:
3.5 คะแนน
